sirepo/api.py

```diff
@@ -0,0 +1,15 @@
+class APIRequest:
+    def __init__(self):
```

Comment:
If there's nothing in `__init__` then don't have one so that the superclass init is automatically called.

Reply:
3ccfbc6e956420f7da6fc23176cdebb7ae57cfc9